packages/drivers/odsp-driver/package.json

```diff
@@ -13,8 +13,9 @@
   "scripts": {
     "build": "npm run build:genver && concurrently npm:build:compile npm:lint",
     "build:commonjs": "npm run tsc && npm run build:test",
-    "build:compile": "concurrently npm:build:commonjs npm:build:esnext",
+    "build:compile": "concurrently npm:build:commonjs npm:build:esnext && npm run build:docs",
```

Comment:
we shouldn't build docs in the compile step. is can be in the build script

Reply:
Good catch; this was a mistake.